src/index.html

```diff
@@ -1,12 +1,541 @@
 <!doctype html>
-<html lang="en">
+<html 
+  lang="en"
+  class="default">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="default__body">
+    <header class="default__header header">
+      <a 
+        href="#"
+        class="header__link">
+        <img 
+          src="./images/logo.png" 
+          alt="logo image"
+          class="header__image"
+        >
+      </a>
+      
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a> 
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="default__main section">
+      <div class="section__1">
```

Comment:
As I said before you don't need to wrap this section into small ones. Let's imagine there will be more or less than 8 cards. What will it be in this case? The layout will work? Just use flex and felx-wrap property for the main container. And calculate the correct width.

Reply:
fixed it, now have only one section with 8 blocks inside. As i mentioned earlier, i don't have reference how design of cards should looks like if there is more or less than 8 cards. So i choose to use `justify-content: flex-start;` so now if there is <8 cards the last one will disappear, if >8 new card will appear from the beginning of new row. But maybe `justify-content: space-between`; will look nicer. Any option can be applied.

Reply:
Also i add all the paddings, but i still can get the problem with fonts. It looks like there is a problem with fonts in entire document, but cards tests are ok, and i can't get the problem. Checked everything several times.